Makefile

```diff
@@ -86,4 +86,3 @@ clean_http_parser:
 
 pre-build:
 	git config core.hooksPath hooks
--include main.c.d
```

Comment:
Why did you remove dependency description?

Reply:
Not need anymore.